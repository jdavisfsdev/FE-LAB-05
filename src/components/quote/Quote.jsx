import React from 'react';
import style from './Quote.module.css';

export function Quote({ quote }) {
  return (
    <div className={style.quoteDisplay}>
      <h2>{JSON.stringify(quote)}</h2>
    </div>
  );
}
