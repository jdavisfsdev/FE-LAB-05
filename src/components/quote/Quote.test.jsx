import React from 'react';
import { render } from '@testing-library/react';
import Quote from './Quote';

describe('Quote component Snaparoo', () => {
  it('renders quote area', () => {
    return;
  });
});
