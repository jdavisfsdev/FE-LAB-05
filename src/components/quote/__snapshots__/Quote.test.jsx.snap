// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote component Snaparoo renders quote area 1`] = `
<DocumentFragment>
  <div
    class="quoteDisplay"
  >
    <h1>
      Random Rick and Morty Character
    </h1>
    <h2 />
  </div>
</DocumentFragment>
`;
